sklearn/tests/test_common.py

```diff
@@ -367,13 +367,11 @@ def _estimators_that_predict_in_fit():
 def test_pandas_column_name_consistency(estimator):
     _set_checking_parameters(estimator)
     with ignore_warnings(category=(FutureWarning)):
-        with pytest.warns(None) as record:
```

Comment:
This should be replaced by ``with warnings.catch_warnings(record=True) as record:``

Reply:
@jeremiedbb thanks for your review. I have made the necessary changes.